keyboards/lets_split/keymaps/lbibass/keymap.c

```diff
@@ -0,0 +1,200 @@
+#include "lets_split.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _WINDOWS 1
+#define _LOWER 4
+#define _RAISE 3
+#define _ADJUST 5
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  WINDOWS,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+enum {
+  CT_media=1
+};
+void user_fn_on_dance_finished (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 2) {
+    register_code (KC_MFFD);
+  }
+  if(state->count == 3){
+    register_code(KC_MRWD);
+  }
+  else {
+    register_code (KC_MPLY);
+  }
+}
+void user_fn_on_dance_reset(qk_tap_dance_state_t *state, void *user_data)
+  {
+   if (state->count == 2)
+   {
+     unregister_code(KC_MFFD);
+   }
+    if (state->count == 3)
+    {
+     unregister_code(KC_MRWD);
+   }
+   else
+    {
+      unregister_code(KC_MPLY);
+   }
+  }
+  qk_tap_dance_action_t tap_dance_actions[] = {
+   [CT_media] = ACTION_TAP_DANCE_FN_ADVANCED(NULL, user_fn_on_dance_finished, user_fn_on_dance_reset)
+  };
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31